ceph/ceph.py

```diff
@@ -550,8 +547,8 @@ def check_health(self, client=None, timeout=300):
         all_osds = int(match.group(1))
         up_osds = int(match.group(2))
         in_osds = int(match.group(3))
-        if self.ceph_demon_stat['osd'] != all_osds:
-            logger.error("Not all osd's are up. Actual: %s / Expected: %s" % (all_osds, self.ceph_demon_stat['osd']))
+        if up_osds != all_osds:
```

Comment:
Is it ok for every suite?

Reply:
Yup, it will compare with parsed output of ceph status command, instead of OSD count from ceph_demon_stat.